packages/@ember/-internals/glimmer/lib/components/input.ts

```diff
@@ -11,61 +11,39 @@ let Input: any;
 
 if (EMBER_GLIMMER_ANGLE_BRACKET_BUILT_INS) {
   /**
-    The `{{input}}` helper lets you create an HTML `<input />` component.
-    It causes a `TextField` component to be rendered.  For more info,
-    see the [TextField](/api/ember/release/classes/TextField) docs and
-    the [templates guide](https://guides.emberjs.com/release/templates/input-helpers/).
+    See [Ember.Templates.components.Input](/api/ember/release/classes/Ember.Templates.components/methods/Input?anchor=Input).
 
-    ```handlebars
-    {{input value="987"}}
-    ```
+    @method input
+    @for Ember.Templates.helpers
+    @param {Hash} options
+    @public
+   */
 
-    renders as:
+  /**
+    The `Input` component lets you create an HTML `<input>` element.
 
-    ```HTML
-    <input type="text" value="987" />
+    ```handlebars
+    <Input @value="987" />
     ```
 
+    creates an `<input>` element with `type="text"` and value set to 987.
+
     ### Text field
 
-    If no `type` option is specified, a default of type 'text' is used.
-    Many of the standard HTML attributes may be passed to this helper.
-    <table>
-      <tr><td>`readonly`</td><td>`required`</td><td>`autofocus`</td></tr>
-      <tr><td>`value`</td><td>`placeholder`</td><td>`disabled`</td></tr>
-      <tr><td>`size`</td><td>`tabindex`</td><td>`maxlength`</td></tr>
-      <tr><td>`name`</td><td>`min`</td><td>`max`</td></tr>
-      <tr><td>`pattern`</td><td>`accept`</td><td>`autocomplete`</td></tr>
-      <tr><td>`autosave`</td><td>`formaction`</td><td>`formenctype`</td></tr>
-      <tr><td>`formmethod`</td><td>`formnovalidate`</td><td>`formtarget`</td></tr>
-      <tr><td>`height`</td><td>`inputmode`</td><td>`multiple`</td></tr>
-      <tr><td>`step`</td><td>`width`</td><td>`form`</td></tr>
-      <tr><td>`selectionDirection`</td><td>`spellcheck`</td><td>&nbsp;</td></tr>
-    </table>
-    When set to a quoted string, these values will be directly applied to the HTML
-    element. When left unquoted, these values will be bound to a property on the
-    template's current rendering context (most typically a controller instance).
-    A very common use of this helper is to bind the `value` of an input to an Object's attribute:
+    If no `type` argument is specified, a default of type 'text' is used.
```

Comment:
do we want to say `@type` here?

Reply:
You suggested to me to remove the `@`s 😛